docs/content-server-api/CONTENT_SERVER_API.md

```diff
@@ -0,0 +1,23 @@
+# Content Server API
+
+## Endpoints
+
+* [Audit] (audit/:type/:entityId/get.md) : `GET /audit/:type/:entityId`
+* [Challenge] (challenge/get.md) : `GET /challenge`
+* [Contents per hashId] (contents/hashId/get.md) : `GET /contents/:hashId`
+* [Available Content] (contents/available-content/get.md) : `GET /available-content`
```

Comment:
Are these supposed to be links?

Reply:
The idea of this schema is to have one file per method, in the future we should have all the endpoints+methods documented. Do you think that it's a good idea to create the files as empty and put "Work in progress" there?

Reply:
I see 2 options:
* Have links for all endpoints, and have a `Work in progress` like you said
* Have links for only the documented endpoints

I'm fine either way